src/modules/add.js

```diff
@@ -38,6 +38,12 @@ class TodoTask {
     inputList.value = '';
   }
 
+   clearTasks = () => {
+     toDoList = toDoList.filter(() => false);
+     displayList.innerHTML = '';
+     localStorage.removeItem('tasks');
+   };
```

Comment:
- [ ] Kindly make sure to fix your `clearTasks` method in order to execute the expected functionality. At this moment when the user clicks on `clearCompleted` button all tasks are deleted while we only need to delete only the completed task.

  - ### YOUR APP BEHAVIOUR

  ![chrome-capture-2023-6-15](https://github.com/MargaretTechworld/ToDO-Tasks/assets/87197412/4da534d2-80bb-4227-bc2b-e0895791ab49)